api/v1alpha1/dspipeline_types.go

```diff
@@ -44,6 +44,12 @@ type DSPASpec struct {
 	// +kubebuilder:validation:Optional
 	// +kubebuilder:default:="v1"
 	DSPVersion string `json:"dspVersion,omitempty"`
+
+	// InterPodTLS Set to "true" or "false" tls communication between DSPA components. Default to enable Inter-Pod tls by setting to "true". Only supported in DSP V2 and OpenShift.
+	// +kubebuilder:default:=true
+	// +kubebuilder:validation:Optional
+	InterPodTLS *bool `json:"interPodTLS"`
```

Comment:
This term is a little clunky/imprecise.  Suggesting `EncryptInternalTraffic` instead?

Reply:
"TLS" is more precise than "Encrypt" :)

"InterPod" is more precise than "Internal" :) It's a tad clunky though.

PodToPodTLS would be my vote.

Reply:
changed to PodToPodTLS